css-pseudo-classes/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS-Pseudo-Classes</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <div>
+      <h1>Hover Button</h1>
+      <label for="button">
+        <input type="button" id="button" class="button" name="button" value="I'm a button!">
```

Comment:
Input type buttons are best used inside of forms for initiating submission of the form. This could just be a button.

Reply:
![image](https://user-images.githubusercontent.com/76827487/111380693-ecef7d80-8661-11eb-9608-9095efcaa35c.png)
Corrected the code, please review. Thanks.